.github/workflows/tests-ubuntu.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       uses: codecov/codecov-action@v3
       if: ${{ success() }}
       with:
-        fail_ci_if_error: true
+        fail_ci_if_error: false
```

Comment:
I disabled it to fix the failing jobs, which were failing at codecov step. And indeed, the jobs passed after this change. But after this change the codecov step was successful (it seems there are some random failures), so I don't think they passed because of the change :)